content/blog/2021-11-23-mlops-architecture-playground.md

```diff
@@ -0,0 +1,118 @@
+---
+title: A Picture worth 1000 Words - The New MLOps Architecture Playground
+date: 2021-11-23
+description: >
+  Introducing a new way the MLOps Open-Source Community can contribute to the
+  greater good.
+descriptionLong: >
+  Having an issue conceptualizing and communicating your MLOps workflows?  Let's
+  work together to solve the problems we all face in the MLOps Architecture
+  Playground.
+picture: 2021-11-02/playground.png
+pictureComment: Join Us on the MLOps Architecture Playground
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/blog-mlops-architecture-playground/934
+tags:
+  - MLOps
+  - Architecture
+  - Reproducibility
+  - DVC
+  - Workflows
+  - Community
+---
+
+## Intro
+
+As the Community Manager at Iterative, I am charged with taking care of and
+serving our Community . I scour the internet ingesting all the amazing content
+our users create, talk to you at Meetups, and have listened to your challenges
+and pain points in calls with you or through Discord. Over this time, some
+things I've noticed in the Community and life in general:
+
+- The best (and most shared) content has good images and diagrams to drive
+  clarity
+- Given the flexibility and hands-off-your-code characteristics of our tools, it
+  can sometimes be challenging for people to conceptualize their workflows
+- diagram images can often be hard to read in a blog post because they take up a
+  lot of space and are illegible when condensed
+- the best solutions and ideas most often come from many places
+- many hands make light work
+- we have an awesome Community of exceptional humans!
+
+So digesting these things and attending other community events where a Miro
+board was used to collaborate and solve problems, I came up with the idea for
+the **MLOps Architecture Playground** in Miro.
+
+![MLOps Architecture Playground by Iterative.ai](/uploads/images/2021-11-23/mlops-playground.png)
+_Join us in the MLOps Archtecture Playground!
+([Source link](https://www.ravirajag.dev/blog/mlops-summary))_
+
+### What?
+
+What this will be is a place for people in the greater MLOps Community to come
```

Comment:
In general, I'd refer to ML engineers or data scientists rather than MLOps Community.

Reply:
I bit more details. These two are our target audience. I don't know how happy these folks would be if you call them MLOps. They might decide the tool is not for them.